File: src/c/problem set 1/lesson 2 - mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int size;
    do
    {
        size = get_int("Size of Pyramid: ");
    }
    while (size < 1);

    for (int row = 1; row <= size; row++)
    {
        for (int spaces = size - row; spaces > 0; spaces--)
        {
            printf(" ");
        }

        for (int column = 1; column <= row; column++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: src/python/problem set 6/lesson 1 - sentimental-mario-less/mario.py
from cs50 import get_int


while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

for row in range(height):
    binary_offset = 1
    blocks = row + binary_offset
    spaces = height - blocks
    print(" " * spaces, "#" * blocks, sep="")
